client/controllers/electionCtrl.js

```diff
@@ -89,6 +89,19 @@ module.exports = [
       $scope.priorities = $scope.priorities.filter((a) => a._id !== id);
     };
 
+    /**
+     * Toggle the chosen alternative. Makes sense only when voting on a NORMAL election not using
+     * stv.
+     * @param {Object} alternative
+     */
```

Comment:
This is not the best comment I've seen.
I would say something like:
> Toggle alternative is only used for NORMAL elections. If they press the selected alternative it will toggle the selection. If they press another alternative it will deselect the current, and reselect the pressed one.